client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -209,6 +209,36 @@ const PluginMeta = React.createClass( {
 		}
 	},
 
+	maybeDisplayAtUnsupportedNotice() {
```

Comment:
I read the `At` as `at`, so maybe we can just call this `maybeDisplayUnsupportedNotice()`?